src/store/Controller.cc

```diff
@@ -398,7 +398,7 @@ Store::Controller::memoryOut(StoreEntry &e, const bool preserveSwappable)
     else
         keepInLocalMemory = keepForLocalMemoryCache(e);
 
-    debugs(20, 7, HERE << "keepInLocalMemory: " << keepInLocalMemory);
+    debugs(20, 7, "keepInLocalMemory: " << keepInLocalMemory);
```

Comment:
> keepInLocalMemory here needs an = not a ':' since it is now in the place of function/method name, but is a variable.

You are inventing rules on the fly. A lot of Squid code uses `:` instead of `=` for variables. I am not against establishing more debugs() rules, but this PR is not the place to do it.